local-setup/docker-compose.yml

```diff
@@ -116,8 +116,8 @@ services:
       # - IPFS_API_URL=http://ipfs:5001
       - ML_RPC_URL=http://host.docker.internal:13034 # change to mainnet if needed
       - ML_BATCH_SIZE=10 # change if necessary
-      - 4EVERLAND_API_KEY=5F2R8SK2EQNSNCHSRWIK # only for test
-      - 4EVERLAND_SECRET_KEY=sCGfIdQZfis8YVCXnQP53SL8cPdRxyzjPLh1KYmF # only for test
+      - 4EVERLAND_API_KEY=PKN5TUJTPYNSSOGC3JW9 # only for test
```

Comment:
We must move all secrets out of the configuration file, an environment variable is good option.